xbmc/Application.cpp

```diff
@@ -5052,6 +4910,80 @@ void CApplication::SetLoggingIn(bool switchingProfiles)
   m_saveSkinOnUnloading = !switchingProfiles;
 }
 
+void CApplication::print_startup_log()
+{
+  CLog::Log(LOGINFO, "-----------------------------------------------------------------------");
+  CLog::Log(LOGINFO, "Starting %s (%s). Platform: %s %s %d-bit", CSysInfo::GetAppName().c_str(),
```

Comment:
Please update to the current log standards using `{}`
For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)